Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN apt-get update \
   uuid-runtime \
   tzdata \
   jq \
-  && if [[ "$(. /etc/lsb-release; echo "${DISTRIB_RELEASE}" | cut -d. -f1)" -ge 18 ]]; then apt-get install gh gpg skopeo; fi \
+  && if [[ "$(. /etc/lsb-release; echo "${DISTRIB_RELEASE}" | cut -d. -f1)" -ge 18 ]]; then apt-get install -y gh gpg skopeo; fi \
```

Comment:
I think the `-y` here is needed. This line has included `gpg` since https://github.com/Homebrew/brew/commit/2acabcae0d7da786d76c96a75290ded819254778, which is is in the 4.0.6 tag, but the container still doesn't have `gpg` on it. (Not sure how our publish workflows have worked though?)

Reply:
`-y` isn't needed if only one thing is installed with no dependencies. Probably worked because of that.